site/src/components/Resources/AgentVersion.tsx

```diff
@@ -44,19 +50,52 @@ export const AgentVersion: FC<{
         onOpen={() => setIsOpen(true)}
         onClose={() => setIsOpen(false)}
       >
-        <HelpTooltipTitle>Agent Outdated</HelpTooltipTitle>
-        <HelpTooltipText>
-          This agent is an older version than the Coder server. This can happen
-          after you update Coder with running workspaces. To fix this, you can
-          stop and start the workspace.
-        </HelpTooltipText>
+        <HelpTooltipContext.Provider
```

Comment:
We have a tooltips folder; `src/components/Tooltips`. This might be nice to add there.